src/graph.cpp

```diff
@@ -91,15 +95,6 @@ Graph::~Graph() {
   open_graph = false;
 }
 
-void Graph::update(GraphUpdate upd) {
```

Comment:
does inlining this make us faster?

Reply:
Just ran a test. Seems to mostly be a difference for small graphs.

| Dataset | Inlining update | No inlining |
|-|-|-|
| kron13 | 5.27 | 5.17 |
| kron15 | 4.80 | 4.80 |